snapcraft/plugins/v1/catkin.py

```diff
@@ -256,6 +260,14 @@ def schema(cls):
 
         schema["required"] = ["source"]
 
+        schema["properties"]["skip-keys"] = {
+            "type": "array",
+            "minitems": 1,
```

Comment:
```suggestion
            "minItems": 1,
```

Capital I. We've made this mistake in a number of places as you can see :blush: .

Reply:
Should I fix the others?

Reply:
Nah, not your problem, we should just have one big PR to fix the existing.

Reply:
ok